WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -262,6 +262,18 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
                     }
                 }
             }
+        } else if ([[url host] isEqualToString:@"faq"]) {
+            if ([HelpshiftUtils isHelpshiftEnabled]) {
+                NSString *faqID = [url lastPathComponent];
+
+                UIViewController *viewController = self.window.rootViewController.presentedViewController ?: self.window.rootViewController;
```

Comment:
That `presentedViewController` can in turn be presenting another VC and the FAQ won't load. This is not even a rare case when dealing with NUX and support:

- Fresh install
- Tap the help button
- Open Help Center or Contact Us (optional)
- Open one of the FAQ URLs from Safari
- App launches but hierarchy doesn't change

The presentation chain then is TabBar -> NUX -> Support -> HelpCenter, so `viewController` would be NUX, which is already presenting.

I think we have two options. We either find the topmost presented controller and present from there, or we make sure that we replace the whole VC hierarchy with what we want. The second is more in line with what deep linking is supposed to be, but if we're only using it for FAQ links, the first is much easier:

```swift
extension UIWindow {
    var topmostPresentedViewController: UIViewController? {
        guard var controller = rootViewController else {
            return nil
        }
        while let presented = controller.presentedViewController {
            controller = presented
        }
        return controller
    }
}
```